client/app/dedicated/server/bandwidth-vrack/order-agora/dedicated-server-bandwidth-agora-vrack.html

```diff
@@ -0,0 +1,58 @@
+<div data-ng-controller="ServerOrderAgoraBandwidthVrackCtrl as ctrl">
+    <div data-wizard
+         data-wizard-on-cancel="resetAction"
+         data-wizard-on-finish="order"
+         data-wizard-title=":: 'server_order_bandwidth_vrack_title' | translate"
+         data-wizard-bread-crumb
+         data-wizard-confirm-button-text=":: 'wizard_pay' | translate">
+
+        <div data-wizard-step
+             data-wizard-step-valid="ctrl.steps[0].isValid()"
+             data-wizard-step-on-load="initFirstStep">
+            <div class="text-center" data-ng-if="ctrl.steps[0].isLoading()">
+                <oui-spinner></oui-spinner>
+                <span data-translate="server_order_bandwidth_vrack_load"></span>
+            </div>
+            <span data-ng-if="ctrl.plans.length === 0" data-translate="server_order_bandwidth_empty_plans"></span>
+            <form data-ng-if="!ctrl.steps[0].isLoading() && ctrl.plans.length > 0">
+                <label class="control-label" data-translate="select_bandwidth_vrack_type_label"></label>
+                <div data-ng-repeat="plan in ctrl.plans track by $index">
+                    <div class="row">
+                        <div class="col-md-9 pr-0">
+                            <oui-radio data-name="bandwidth-plan" data-model="ctrl.model.plan" data-value="plan.planCode">
+                                <span data-translate="{{ 'plan_' + plan.planCode }}"></span>
+                            </oui-radio>
+                        </div>
+                        <div class="col-md-3 pl-0">
+                            <b class="red"><span data-ng-bind-html="plan.prices[2].price.text"></span></b>
+                            <i class="small"><span data-translate="plan_price_per_month"></span></i>
+                        </div>
+                    </div>
+                </div>
+                <small data-translate="server_order_bandwidth_declaration"></small>
```

Comment:
`small` is used for legal and copyright texts (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small), please use a `p` with a class instead